src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -166,7 +166,7 @@ Temporarily using the Windows Terminal default settings.
     <value>About</value>
   </data>
   <data name="VersionLabelText" xml:space="preserve">
-    <value>Version:</value>
+    <value>WT Version:</value>

```

Comment:
I think I'd personally leave this unchanged, but maybe that's a personal stylistic choice. @DHowett-MSFT/@cinnamon-msft do you have thoughts here?

(TBH you don't need to change this back unless someone else agrees with me here)